### PR TITLE
Refactor/profile modal

### PR DESCRIPTION
**Pull request by theketan26**
## Description

When the user clicks on the profile icon in the sidebar, the user will be able to see the user name, user email address, and sign-out button.

## What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [x] 🎨 Style
- [x] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build

## Have you tested locally?

- [ ] 👍 yes
- [ ] 🙅 no, because I am lazy

## Added tests?

- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md

### CODE REVIEW

**Comment by punitkashyup** [comment]
Please attach the issue link

File: src/packages/@app/styles/style.scss [hunk 1]
```diff
@@ -354,7 +356,8 @@ $theme-colors: map-merge(
     "dropdownContainer": $sparrow-dropdown-container,
     "dropdownOptionHover": $sparrow-dropdown-option-hover,
     "dropdownOptionActive": $sparrow-dropdown-option-active,
-    "saveButtonLine": $sparrow-send-button-line
+    "saveButtonLine": $sparrow-send-button-line,
+    "sparrowProfileModalBg": $sparrow-profile-modal-bg
   )
```
Comment (itsmdasifraza) [inline src/packages/@app/styles/style.scss:361]: use revised format for creating css variables

**Comment by punitkashyup** [comment]
Please attach the issue link

File: src/packages/@common/components/sidebar/SignOutIcon.svelte [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+<svg
+  width={size}
+  height={size * 0.8333}
+  viewBox="0 0 12 10"
+  fill="none"
+  xmlns="http://www.w3.org/2000/svg"
+>
+  <path
+    d="M11.1368 5.47087C11.3973 5.21046 11.3973 4.78754 11.1368 4.52712L8.47018 1.86046C8.20977 1.60004 7.78685 1.60004 7.52643 1.86046C7.26602 2.12087 7.26602 2.54379 7.52643 2.80421L9.0556 4.33337H4.66602C4.29727 4.33337 3.99935 4.63129 3.99935 5.00004C3.99935 5.36879 4.29727 5.66671 4.66602 5.66671H9.0556L7.52643 7.19587C7.26602 7.45629 7.26602 7.87921 7.52643 8.13962C7.78685 8.40004 8.20977 8.40004 8.47018 8.13962L11.1368 5.47296V5.47087ZM3.99935 1.66671C4.3681 1.66671 4.66602 1.36879 4.66602 1.00004C4.66602 0.631291 4.3681 0.333374 3.99935 0.333374H2.66602C1.56185 0.333374 0.666016 1.22921 0.666016 2.33337V7.66671C0.666016 8.77087 1.56185 9.66671 2.66602 9.66671H3.99935C4.3681 9.66671 4.66602 9.36879 4.66602 9.00004C4.66602 8.63129 4.3681 8.33337 3.99935 8.33337H2.66602C2.29727 8.33337 1.99935 8.03546 1.99935 7.66671V2.33337C1.99935 1.96462 2.29727 1.66671 2.66602 1.66671H3.99935Z"
```
Comment (itsmdasifraza) [inline src/packages/@common/components/sidebar/SignOutIcon.svelte:13]: move this file to library/icons

**Comment by punitkashyup** [comment]
Please attach the issue link

**Comment by punitkashyup** [comment]
Please attach the issue link

**Pull request merged**